src/contracts/interfaces/IVoteWeigher.sol

```diff
@@ -1,25 +1,92 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity =0.8.12;
 
+import "../interfaces/IStrategyManager.sol";
+import "../interfaces/IServiceManager.sol";
+import "../interfaces/ISlasher.sol";
+import "../interfaces/IDelegationManager.sol";
+
 /**
  * @title Interface for a `VoteWeigher`-type contract.
  * @author Layr Labs, Inc.
  * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
  * @notice Note that `NUMBER_OF_QUORUMS` is expected to remain constant, as suggested by its uppercase formatting.
  */
 interface IVoteWeigher {
+    /**
+     * @notice In weighing a particular strategy, the amount of underlying asset for that strategy is
+     * multiplied by its multiplier, then divided by WEIGHTING_DIVISOR
+     */
+    struct StrategyAndWeightingMultiplier {
+        IStrategy strategy;
+        uint96 multiplier;
+    }
+
+    /// @notice Constant used as a divisor in calculating weights.
+    function WEIGHTING_DIVISOR() external pure returns (uint256);
+
+    /// @notice Returns the strategy manager contract.
+    function strategyManager() external view returns (IStrategyManager);
+    /// @notice Returns the stake registry contract.
+    function slasher() external view returns (ISlasher);
```

Comment:
this comment must be wrong.
Also might want to note whether these contracts are AVS-specific or EigenLayer-level (or maybe some other shared level).

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/926aabdfc303677f87da99a8cd5ba845d491a18f